venia-integration-tests/cypress.config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "baseUrl": "https://develop.pwa-venia.com/",
+    "baseUrl": "https://magento-venia-concept-7nbzp.local.pwadev:8708/",
```

Comment:
@DiegoBezerra89 Please revert it to develop.

Reply:
Sorry...